hbase-server/src/test/java/org/apache/hadoop/hbase/client/TestSplitOrMergeAtTableLevel.java

```diff
@@ -217,14 +218,15 @@ private void tryMergeAndEnsureItFails(final TableName tableName) throws Exceptio
     byte[] nameOfRegionB = regions.get(1).getEncodedNameAsBytes();
 
     // check and ensure that region do not get merged
-    Future<?> f = admin.mergeRegionsAsync(nameOfRegionA, nameOfRegionB, true);
+    Future<?> f = admin.mergeRegionsAsync(new byte[][] { nameOfRegionA, nameOfRegionB }, true);
```

Comment:
Why we need this change? Because the method which takes two byte[] is deprecated?

Reply:
Hi Duo, yes saw this deprecated usage in the test. mentioned same in description. let me know if should revert

Reply:
No problem. Just want to confirm.